components/Input/index.test.jsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+
+import Input from '.';
+
+describe('Input', () => {
+  it('renders a input box', () => {
+    const props = {
+      id: 'email',
+      type: 'email',
+      required: true,
+      dataTestId: 'email_input',
+    };
+
+    render(<Input {...props} />);
+
+    const input = screen.getByTestId(props.dataTestId);
+
+    expect(input).toBeVisible();
+    expect(input).toHaveAttribute('type', props.type);
+  });
```

Comment:
Should we add the case when passing className to ensure that the rendered component has that className?

Reply:
added on d129258d877c1e23ba31313cc0390ad9627eb192